parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -261,6 +258,7 @@ write_files:
     ExecStartPost=/sbin/iptables -P FORWARD ACCEPT
     #EOF
 
+{{if RequiresDocker}}
```

Comment:
I think we need an `{{end}}`` corresponding to this?

Reply:
L284 I think -- diff cuts it off